src/main/java/io/openliberty/tools/intellij/lsp4jakarta/lsp4ij/codeAction/proposal/quickfix/RemoveParamAnnotationQuickFix.java

```diff
@@ -14,23 +14,31 @@
 
  import com.intellij.psi.*;
  import com.intellij.psi.util.PsiTreeUtil;
+ import io.openliberty.tools.intellij.lsp4jakarta.lsp4ij.JDTUtils;
  import io.openliberty.tools.intellij.lsp4jakarta.lsp4ij.Messages;
  import io.openliberty.tools.intellij.lsp4jakarta.lsp4ij.codeAction.proposal.RemoveAnnotationsProposal;
+ import io.openliberty.tools.intellij.lsp4mp4ij.psi.core.java.codeaction.IJavaCodeActionParticipant;
  import io.openliberty.tools.intellij.lsp4mp4ij.psi.core.java.codeaction.JavaCodeActionContext;
+ import io.openliberty.tools.intellij.lsp4mp4ij.psi.core.java.codeaction.JavaCodeActionResolveContext;
  import org.eclipse.lsp4j.CodeAction;
  import org.eclipse.lsp4j.Diagnostic;
+ import org.eclipse.lsp4j.WorkspaceEdit;
+ import org.eclipse.lsp4mp.commons.CodeActionResolveData;
 
- import java.util.ArrayList;
- import java.util.Arrays;
- import java.util.List;
+ import java.util.*;
+ import java.util.logging.Level;
+ import java.util.logging.Logger;
 
  /**
   * QuickFix for removing parameter annotations
   */
-public class RemoveParamAnnotationQuickFix {
+public abstract class RemoveParamAnnotationQuickFix implements IJavaCodeActionParticipant {
 
 	private final String[] annotations;
 	
+    private static final Logger LOGGER = Logger.getLogger(RemoveParamAnnotationQuickFix.class.getName());
+    private static final String ANNOTATION_TO_REMOVE = "annotationsToRemove";
+    private static final String INDEX_KEY = "paramIndex";
```

Comment:
Add a space before constructor definition.

Reply:
FYI for future PRs, it is better practice to do reformatting of files in a separate PR that is solely for reformatting and contains no other functional changes. When reformatting is mixed with functional changes, it becomes difficult to see what the functional changes actually are.